core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1689,8 +1694,8 @@ class KafkaApis(val requestChannel: RequestChannel,
           (txnCoordinator.partitionFor(key), TRANSACTION_STATE_TOPIC_NAME)
 
         case CoordinatorType.SHARE =>
-          // When share coordinator support is implemented in KIP-932, a proper check will go here
-          return (Errors.COORDINATOR_NOT_AVAILABLE, Node.noNode)
+          // None check already done above
```

Comment:
Sharecoordinator is an Optional. The line below the comment is directly calling Sharecoordinator.get, because the case shareCoordinator.isEmpty check is already done in line 1703